src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String RESULT = "result";
+    public static final String SEPARATOR = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        readDataFromFile(fromFileName);
+        createReport(toFileName, readDataFromFile(fromFileName));
+    }
+
+    public String readDataFromFile(String fileName) {
+        int supply = 0;
+        int buy = 0;
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName));
+            StringBuilder stringBuilder = new StringBuilder();
+            String stringLine = bufferedReader.readLine();
+            while (stringLine != null) {
+                if (stringLine.split(SEPARATOR)[0].equals(SUPPLY)) {
+                    supply += Integer.parseInt(stringLine.split(SEPARATOR)[1]);
+                } else {
+                    buy += Integer.parseInt(stringLine.split(SEPARATOR)[1]);
+                }
+                stringLine = bufferedReader.readLine();
+            }
+            bufferedReader.close();
```

Comment:
use close here is a bad practice because if we will have some exception our resource won't close